src/types/timezone.jl

```diff
@@ -105,3 +105,10 @@ function istimezone(str::AbstractString, mask::Class=Class(:DEFAULT))
 
     return tz !== nothing && mask & class != Class(:NONE)
 end
+
+# After `broadcastable` was defined but before this was addressed in the `Dates` stdlib
+# - Broadcastable introduction: https://github.com/JuliaLang/julia/pull/26601
+# - Fixed in Dates: https://github.com/JuliaLang/julia/pull/30159
+if v"0.7.0-DEV.4743" <= VERSION <= v"1.2.0-DEV.114"
+    Base.broadcastable(tz::TimeZone) = Ref(tz)
```

Comment:
This will overwrite the base definition on v1.1

Reply:
I'll update this rule to skip the back-ported revisions

Reply:
Addressed to avoid the back ported changes in revision 6778bef3ad